app/javascript/react/utils/ThresholdsValues.ts

```diff
@@ -0,0 +1,7 @@
+export interface ThresholdsValues {
```

Comment:
I think that now we have this interface in types folder, could you remove this one if that's correct?